hypervisor/debug/profiling.c

```diff
@@ -77,27 +78,138 @@ static void profiling_pmi_handler(__unused unsigned int irq, __unused void *data
 /*
  * Generate VM info list
  */
-int32_t profiling_vm_list_info(__unused struct vm *vm, __unused uint64_t addr)
+int32_t profiling_vm_list_info(struct vm *vm, uint64_t addr)
 {
-	/* to be implemented */
+	struct vm *tmp_vm;
+	struct vcpu *vcpu;
+	int32_t vm_idx;
+	uint16_t i, j;
+	struct profiling_vm_info_list vm_info_list;
+
+	(void)memset((void *)&vm_info_list, 0U, sizeof(vm_info_list));
+
+	dev_dbg(ACRN_DBG_PROFILING, "%s: entering", __func__);
+
+	if (copy_from_gpa(vm, &vm_info_list, addr, sizeof(vm_info_list)) != 0) {
```

Comment:
0U

Reply:
Return type for copy_from_gpa is int. In that case isn't using '0'  correct